src/styles/product.scss

```
@@ -0,0 +1,102 @@
+$accent-color-blue: #00acdc;
+$sub-text-color-grey: #616070;
+
+.product {
+  display: flex;
+  justify-content: center;
+
+  &__wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 50px 0;
+  }
+}
+
+.product-card {
+  width: 200px;
+  padding: 32px 16px 19px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-right: 48px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-child(n + 5) {
+    margin-top: 42px;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    color: #{$sub-text-color-grey};
+    margin-bottom: 17px;
+  }
+
+  &__rating {
+    justify-content: space-between;
+    margin-bottom: 22px;
+    display: flex;
+    font-size: 10px;
+  }
+
+  &__reviews {
+    line-height: 18px;
+  }
```

Font-weight may help you to display this text correct in tests
```suggestion
  &__reviews {
    line-height: 18px;
    font-weight: 400;
  }
```